synapse/lib/layer.py

```diff
@@ -1161,6 +1161,14 @@ async def _initLayerStorage(self):
         self.onfini(self.nodeeditslab)
         self.onfini(self.dataslab)
 
+        # various properties/tags/data about a node
+        # 00 - formvalu (buid, 00) = (form, valu, stortype)
+        # 01 - property (buid, 01, penc) = (valu, stortype)
+        # 02 - tag (buid, 01, tenc) = valu
+        # 09 - form
+        # 0a - form tombstone (buid, 0a) = <editinfo>
+        # 0b - prop tombstone (buid, 0b, penc) = <editinfo>
+        # 0c - tag tombstone (buid, 0c, tenc) = <editinfo>
```

Comment:
Other things to consider off hand include the following:

Tag Props
Light Edges
Node Data

Reply:
yep thanks, was def planning to include them eventually :)